uaainvite/templates/email/body.html

```diff
@@ -7,71 +7,35 @@
     <body>
         <p>Hello!</p>
         <p>
-            admin has invited you to join the
-            closed pilot of
-            <a href="https://cloud.gov">{{branding.company_name}}</a>
+            You've been invited to join 
+            <a href="https://cloud.gov">{{branding.company_name}}</a>.
```

Comment:
That makes sense to me! I'm not sure the best way to implement that in this template - feel free to go ahead and put another commit in that does that!